resources/views/authors/show.blade.php

```diff
@@ -42,26 +73,15 @@
                 <td class="px-6 py-4 whitespace-no-wrap border-b border-gray-200 flex justify-between">
                     <div>
                         <div class="text-md leading-5 text-gray-900">
-                            {{ $pr->title }}
-                            @if($pr->isToday())
-                            <span class="px-2 inline-flex text-xs leading-5 font-semibold rounded-full bg-purple-100 text-purple-800">
-                                Today
-                            </span>
-                            @endif
-                        </div>
-                        <div class="text-sm leading-5 text-gray-500">{{ $pr->pr_merged_at->format('Y-m-d H:i') }}</div>
-                    </div>
-                </td>
-                <td class="px-6 py-4 whitespace-no-wrap border-b border-gray-200">
-                    <div class="flex items-center">
-                        <div class="flex-shrink-0 h-10 w-10">
-                            <img class="h-10 w-10 rounded-full" src="{{ $pr->author->photo ?? '#' }}" alt="{{ $pr->author->name ?? '-' }}" />
-                        </div>
-                        <div class="ml-4">
-                            <a class="text-sm leading-5 font-medium text-gray-600" href="{{ $pr->author->url() }}">
-                                {{ $pr->author->name ?? '-' }}
```

Comment:
we need to keep this as it is

Reply:
Okay but I think adding today near the data is it makes more sense

Reply:
also  no need to show author details in each row because It shown in the top op the page